just-pandas-things.ipynb

```diff
@@ -163,7 +163,7 @@
     }
    ],
    "source": [
-    "import pandas as pd\n",
+    "import numpy as np, pandas as pd\n",
```

Comment:
let's import `np` only when it's used (e.g. in your added section on merge bugs)

Reply:
Done in PR#9